atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/CellsSweeper.java

```diff
@@ -82,9 +82,16 @@ public void sweepCells(
             Multimap<Cell, Long> cellTsPairsToSweep,
             Collection<Cell> sentinelsToAdd) {
         if (cellTsPairsToSweep.isEmpty()) {
```

Comment:
Quick check on correctness here: this implies if `cellTsPairsToSweep` is empty but `sentinelsToAdd` is not then we don't do anything.  Is this the correct behaviour?  Can this case ever arise?

Reply:
I don't think this case should ever happen.

Reply:
Confirmed:
- this is only called from `sweepBatch` in `SweepTaskRunner`
- in that method, this is only called if the batch either has size `>= deleteBatchSize` (mid-loop) which is required to be positive, or if it's non-empty (at the end).